components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2877,6 +2825,10 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         }
 
         const team = await this.teamDB.createTeam(user.id, name);
+        // create a cost center
+        await this.usageService.getCostCenter({
```

Comment:
What's the reason for this addition?

Reply:
To create the cost center immediately and not lazily which is harder to reason about